app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -2125,15 +2125,15 @@ class PluginApi {
    * Support for setting a Sidebar panel.
    */
   setSidebarPanel(name) {
-    this._lookupContainer("service:sidebar-state").setPanel(name);
+    this._lookupContainer("service:sidebar-state")?.setPanel(name);
```

Comment:
This and following methods get called during the route's deactivation and at that moment service is null. That lead to errors in tests, so I've added safe navigation here.